sycl-fusion/test/internalization/promote-private-vec.ll

```diff
@@ -30,10 +30,10 @@ declare spir_func void @__itt_offload_wi_start_wrapper() #3
 declare spir_func void @__itt_offload_wi_finish_wrapper() #3
 
 ; Function Attrs: noinline nounwind
-declare  spir_func void @__itt_offload_wi_finish_stub(i64 addrspace(4)* %group_id, i64 %wi_id) #4
+declare  spir_func void @__itt_offload_wi_finish_stub(ptr addrspace(4) %group_id, i64 %wi_id) #4
 
 ; Function Attrs: noinline nounwind
-declare spir_func void @__itt_offload_wi_start_stub(i64 addrspace(4)* %group_id, i64 %wi_id, i32 %wg_size) #4
+declare spir_func void @__itt_offload_wi_start_stub(ptr addrspace(4) %group_id, i64 %wi_id, i32 %wg_size) #4
 
 define spir_kernel void @fused_0(%0 addrspace(1)* align 16 %KernelOne_accTmp, %1* byval(%1) align 8 %KernelOne_accTmp3, %0 addrspace(1)* align 16 %KernelOne_accIn1, %1* byval(%1) align 8 %KernelOne_accIn16, %0 addrspace(1)* align 16 %KernelOne_accIn2, %0 addrspace(1)* align 16 %KernelTwo_accOut, %0 addrspace(1)* align 16 %KernelTwo_accIn3) !kernel_arg_addr_space !12 !kernel_arg_access_qual !13 !kernel_arg_type !14 !kernel_arg_type_qual !15 !kernel_arg_base_type !14 !kernel_arg_name !16 !sycl.kernel.promote !17 !sycl.kernel.promote.localsize !18 {
```

Comment:
Also, is there any reason the typed pointers in this function were not replaced with opaque pointers?